04-Mining-Protocol.md

```diff
@@ -407,25 +407,26 @@ The server provides an updated mining job to the client through a standard chann
 If the `future_job` field is set to False, the client MUST start to mine on the new job as soon as possible after receiving this message.
 
 ```
-+-------------+-----------+--------------------------------------------------------------------------------------------+
-| Field Name  | Data Type | Description                                                                                |
-+-------------+-----------+--------------------------------------------------------------------------------------------+
-| channel_id  | U32       | Channel identifier, this must be a standard channel                                        |
-+-------------+-----------+--------------------------------------------------------------------------------------------+
-| job_id      | U32       | Identifier of the job as provided by NewMiningJob or NewExtendedMiningJob message          |
-+-------------+-----------+--------------------------------------------------------------------------------------------+
-| future_job  | BOOL      | True if the job is intended for a future SetNewPrevHash message sent on this channel. If   |
-|             |           | False, the job relates to the last sent SetNewPrevHash message on the channel and the      |
-|             |           | miner should start to work on the job immediately.                                         |
-+-------------+-----------+--------------------------------------------------------------------------------------------+
++----------------+---------------+-------------------------------------------------------------------------------------+
+| Field Name     | Data Type     | Description                                                                         |
++----------------+---------------+-------------------------------------------------------------------------------------+
+| channel_id     | U32           | Channel identifier, this must be a standard channel                                 |
++----------------+---------------+-------------------------------------------------------------------------------------+
+| job_id         | U32           | Identifier of the job as provided by NewMiningJob or NewExtendedMiningJob message   |
++----------------+---------------+-------------------------------------------------------------------------------------+
+| starting_ntime | OPTION[u32]   | Empty if the job is intended for a future SetNewPrevHash message sent on this       |
```

Comment:
I prefer the name `min_ntime`, but if you don't like that for any reason, `starting_ntime` is also fine

Reply:
This was @bluematt suggestion. I don't have strong opinion on that.

Reply:
Too many communication channels, because I don't remember talking about this on discord. But whatever, let's keep the the original min_ntime.